target/scala-2.11/scala-space-launcher.js

```diff
@@ -0,0 +1,2 @@
+((typeof global === "object" && global &&
+         global["Object"] === Object) ? global : this)["scalaspace"]["ScalaSpace"]().main();
```

Comment:
The `target/` directory should be `.gitignore`d, and definitely *not* be version-controlled.